addons/mail/models/mail_activity.py

```diff
@@ -181,31 +184,44 @@ def create(self, values):
         values_w_defaults = self.default_get(self._fields.keys())
         values_w_defaults.update(values)
 
+        # Reminder have no summary (for display name) It will add first line of note to summary.
+        if 'res_model_id' not in values:
+            text = html2plaintext(values.get('note', _('Reminder')))
+            values_w_defaults['summary'] = text.strip().replace('*', '').split("\n")[0]
+
         # continue as sudo because activities are somewhat protected
         activity = super(MailActivity, self.sudo()).create(values_w_defaults)
         activity_user = activity.sudo(self.env.user)
-        activity_user._check_access('create')
-        self.env[activity_user.res_model].browse(activity_user.res_id).message_subscribe(partner_ids=[activity_user.user_id.partner_id.id])
-        if activity.date_deadline <= fields.Date.today():
-            self.env['bus.bus'].sendone(
-                (self._cr.dbname, 'res.partner', activity.user_id.partner_id.id),
-                {'type': 'activity_updated', 'activity_created': True})
+        if activity.res_id and activity.res_model:
+            activity_user._check_access('create')
```

Comment:
I think we have to always ensure check_access is called. You should update the method to handle the specific case of reminders.

What I have in mind to include in check access:
 - create, unlink: you can always create your own reminders if you are at least employee
 - write: you can update it but not write on res_id / res_model fields, otherwise you may grant unwanted privileges; same for res_id, you can update only your own reminders
 - read: you can read your own reminders

Please write tests to ensure the behavior is effectively working.